docs/test-surveys/13-survey-react-native-release.md

```diff
@@ -0,0 +1,87 @@
+### UC1: Open the app and choose a city of your choice as a location and the language you are most comfortable with. Explore the app and familiarize yourself with it. Turn off your internet connection and explore the same city and a further on.
+
+#### Describe the steps you took.
+
+*Input box*
+
+### UC2: Choose city "Stadt Regensburg" as your location, if you haven’t done so already. Find out when shops are usually open in Regensburg. Switch the language as soon as you found it.
+
+#### Did you find the information?
+
+* [ ] Yes
+* [ ] No
+
+(The Answer is: "Mo-Sat: 08:00-20:00")
+
+#### Describe the steps you took.
+
+*Input box*
+
+### UC3: Find the same information as in UC2 by using the search icon in the navigation bar. 
+
+The search terms you need are "Einkaufen" (German), "Shopping" (German), "التسوّق" (Arabic/العربية)
+
+#### Did you find the information?
+
+* [ ] Yes
+* [ ] No
+
+### UC4: Choose city "Stadt Regensburg" as your location, if you haven’t done so already. Try to find a job in the "Sprungbrett into work" offer. Also try the "Lehrstellenradar".
+
+Note: You need a stable internet connection.
+
+#### Did you a job offer near Regensburg?
+
+* [ ] Yes
+* [ ] No
+
+### UC5: Turn off your internet connection (WiFi + Data) and repeat usecases UC2-4.
+
+Note: If the app does show you what you require internet then it is not a bug!
```

Comment:
```suggestion
Note: If the app does show you that an internet connection is required then it is not a bug!
```
Also this is not entirely true, is it? Already opened cities/languages should work.